pkg/cli/cli.go

```diff
@@ -723,19 +731,27 @@ func runPullCommand(c *cobra.Command, args []string, flags *CVDRemoteFlags, opts
 }
 
 // Returns empty string if there was no host.
-func promptHostNameSelection(c *command, service client.Service) (string, error) {
-	names, err := hostnames(service)
+func promptSingleHostNameSelection(c *command, service client.Service) (string, error) {
+	sel, err := promptHostNameSelection(c, service, 0)
```

Comment:
Use a constant for the SelectionOption parameter

Reply:
Done.